src/AppBundle/Form/JemarcheReportType.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+
+class JemarcheReportType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+	    ->add('actionType', ChoiceType::class, array(
+            'choices' => array(
+                     'La marche' => 'La marche',
```

Comment:
I think we should put canonical identifiers here instead of classical strings (ie `la-marche` instead of `La marche` to ensure a proper database storage).